.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
1. It makes maintaining a project with PRs and different branches hard because you'll get constant merge conflicts with the lock file.

2. Surely the point of tests is to check it **currently works** rather than worked with some old dependencies. You want to know if something broke straight away.